QRCoder/QRCoder.csproj

```diff
@@ -9,6 +9,8 @@
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
 	<IsTrimmable Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))">true</IsTrimmable>
+	<Nullable>enable</Nullable>
+	<LangVersion>9.0</LangVersion>
```

Comment:
In the future we may want to bump this higher . For now I only bumped as necessary for this PR.